runtime/doc/if_lua.txt

```diff
@@ -588,6 +588,22 @@ vim.schedule({callback})				*vim.schedule()*
         Schedules {callback} to be invoked soon by the main event-loop. Useful
         to avoid |textlock| or other temporary restrictions.
 
+vim.fn.{func}({...})
+        Call vimL function {func} with arguments. {func} can be both builtin
+        functions and user functions. To call autoload functions, use the
+        syntax `vim.fn['some#function']({...})`
+
+        Note: unlike vim.api.|nvim_call_function| this converts values directly
+        between vimL values and lua values. If the vimL function returns a
+        float, it will be representeted directly as a lua number. Both empty
```

Comment:
Typo here with `representeted`

Reply:
will fix it in next PR (if doc PR doesn't do it first :)

Reply:
fixed in https://github.com/neovim/neovim/pull/11277